frame/vint_test.go

```diff
@@ -0,0 +1,135 @@
+package frame
+
+import (
+	"math"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestDecodeVint(t *testing.T) {
```

Comment:
Tests in this file could check behavior for all 9 vint lengths

Reply:
Added.